subjects/mobile-dev/chess/resources/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.19.1-buster AS builder
```

Comment:
this might be simplified to `FROM golang:1.19.1-buster`

Reply:
will close this until we work on it!